### DISCUSSION TITLE
How to show information in tool tip

### DISCUSSION THREAD

**Discussion by Vaibhav2121** [nasa/openmct#4743] [Community Q&A]
I want to show some information in tool tip, when user puts the cursor in front of parameters. 
How to do it

**Comment by jvigliotta** [comment]
Hey @Vaibhav2121 when you say parameters... what are you referring to? Telemetry Data? Is it in a Telemetry Table? LAD Table? Plot? Let us know and we can better answer this question for you!

**Reply by Vaibhav2121** [reply]
With parameters I am referring to telemetry parameters, that comes in left side panel, like  battery voltage, bus voltage, total load in watts etc. If I want to display information like battery type and make in tool tip, how to do it.
